cosmic_ray/modules.py

```diff
@@ -35,6 +36,11 @@ def find_modules(name, excludes=None):
 
     """
     excludes = excludes or []
+    # helps when using bash completion
+    if os.path.exists(name):
+        if name.endswith('.py'):
+            name = name[:-3]
+        name = name.replace('/', '.')
```

Comment:
Can you point me to the right place in the code? I see that cli.py calls find_modules twice and I don't want to repeat the same code twice.